SkietbaanBE/SkietbaanBE/Program.cs

```diff
@@ -14,12 +14,13 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            BuildWebHost(args).Run();
+            Console.Write(Environment.GetEnvironmentVariable("TEMP"));
+           // BuildWebHost(args).Run();
         }
 
-        public static IWebHost BuildWebHost(string[] args) =>
+        /*public static IWebHost BuildWebHost(string[] args) =>
```

Comment:
I explained about having ghost code in the forum. Please follow the instructions